pkg/key/handler.go

```
@@ -15,20 +19,42 @@
 	return &Handler{}
 }
 
+func (h *Handler) GetKeylist(w http.ResponseWriter, r *http.Request) {
+	accept_header := r.Header["Accept"]
+	if slices.Contains(accept_header, "application/json") {
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(storage.GetUniqueIdentities())
+	} else {
+		w.Header().Set("Content-Type", "text/html")
+		htmx.RenderKeyTable(w, storage.GetUniqueIdentities())
```

golangci-lint

Error return value of `htmx.RenderKeyTable` is not checked

(errcheck)